src/python/pants/notes/1.18.x.rst

```diff
@@ -0,0 +1,409 @@
+
+* Add some of the wording from #7913 that wasn't merged in #7924  (#7958)
+  `PR #7958 <https://github.com/pantsbuild/pants/pull/7958>`_
+
+* Crash pailgun client on SIGINT if we haven't received the pid yet (#7924)
+  `PR #7924 <https://github.com/pantsbuild/pants/pull/7924>`_
+
+* Add whitelist of folders that must remain Python 2 compatible (#7941)
+  `PR #7941 <https://github.com/pantsbuild/pants/pull/7941>`_
+
+* Add concurrent flag, which overrides --enable-pantsd (#7944)
+  `PR #7944 <https://github.com/pantsbuild/pants/pull/7944>`_
+
+* Report zipkin spans regardless of goal version. (#7915)
+  `PR #7915 <https://github.com/pantsbuild/pants/pull/7915>`_
+
+* Don't memoize description in process execute memoization (#7916)
+  `PR #7916 <https://github.com/pantsbuild/pants/pull/7916>`_
+
+* Modernize classes to use Python 3 syntax (#7937)
+  `PR #7937 <https://github.com/pantsbuild/pants/pull/7937>`_
+
+* Remove configparser backport (#7939)
+  `PR #7939 <https://github.com/pantsbuild/pants/pull/7939>`_
+
+* Remove `collections.abc` backport (#7940)
+  `PR #7940 <https://github.com/pantsbuild/pants/pull/7940>`_
+
+* Remove `self.assertRegex` backport (#7938)
+  `PR #7938 <https://github.com/pantsbuild/pants/pull/7938>`_
+
+* Add --post-compile-merge-dir option in zinc (#7917)
+  `PR #7917 <https://github.com/pantsbuild/pants/pull/7917>`_
+
+* Remove `builtins` imports (#7934)
+  `PR #7934 <https://github.com/pantsbuild/pants/pull/7934>`_
+
+* Modernize `abc` stdlib usage. (#7928)
+  `PR #7928 <https://github.com/pantsbuild/pants/pull/7928>`_
+
+* Workaround Travis timeout for OSX platforms test (#7919)
+  `PR #7919 <https://github.com/pantsbuild/pants/pull/7919>`_
+
+* Modernize all headers to Python 3 style (#7926)
+  `PR #7926 <https://github.com/pantsbuild/pants/pull/7926>`_
+
+* Kill `AbstractClass` in favor of `abc.ABC{,Meta}`. (#7929)
+  `PR #7929 <https://github.com/pantsbuild/pants/pull/7929>`_
+
+
+1.18.0.dev1 (6/21/2019)
+-----------------------
+
+API Changes
+~~~~~~~~~~~
+
+* No longer release a Python 2 wheel or PEX (#7888)
+  `PR #7888 <https://github.com/pantsbuild/pants/pull/7888>`_
+
+* Bump default Mypy version from 0.670 to 0.710 (#7912)
+  `PR #7912 <https://github.com/pantsbuild/pants/pull/7912>`_
+
+Bugfixes
+~~~~~~~~
+
+* Don't nest output directories in an empty path (#7908)
+  `PR #7908 <https://github.com/pantsbuild/pants/pull/7908>`_
+
+* Migrate python_repl to use ignoring-sigint (#7890)
+  `PR #7890 <https://github.com/pantsbuild/pants/pull/7890>`_
+
+* Use Selectors module for more robust and efficient syscalls (#7882)
+  `Issue #7880 <https://github.com/pantsbuild/pants/issues/7880>`_
+  `PR #7882 <https://github.com/pantsbuild/pants/pull/7882>`_
+
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Set internal compatibility to >= CPython 3.6 and remove ./pants2 (#7925)
+  `PR #7925 <https://github.com/pantsbuild/pants/pull/7925>`_
```

Comment:
Possibly an API change?

Reply:
This was long ago, and only applies to development in the pants repo itself :)